### PR TITLE
Update sanic to 20.12 LTS

### PR DESCRIPTION
**Pull request by ancalita**
**Proposed changes**:
- Fixes https://rasahq.atlassian.net/browse/ATO-204
- Ugrades sanic, rasa-sdk, sanic-cors, adds sanic-testing
- Make necessary code adjustments

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [X] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -200,3 +200,11 @@ optional = true
...
 version = ">=3.8,<3.11"
 extras = [ "tls", "srv",]
+
+# uncomment once branch is updated and remove the last 3 lines
+#[tool.poetry.dev-dependencies.pytest-sanic]
+#git = "https://github.com/RasaHQ/pytest-sanic"
+#branch = "fix_signal_issue"
+[tool.poetry.dev-dependencies.pytest-sanic]
+git = "https://github.com/wochinge/pytest-sanic"
+branch = "fix-signal-issue"
```
Comment (melindaloubser1) [inline pyproject.toml:210]: Can we switch to the official lib that Tobi mentioned here https://github.com/yunstanford/pytest-sanic/pull/63#issuecomment-1172429499?

Comment (ancalita) [reply pyproject.toml:210]: That's what I was trying to do on Friday afternoon before our discussion but it didn't work out, I can try again today.

Comment (melindaloubser1) [reply pyproject.toml:210]: Gotcha, if that doesn't work out easily enough, let's just switch this URL to the RasaHQ one

Comment (ancalita) [reply pyproject.toml:210]: I left a comment on your PR to remove those 2 extra lines because it causes error, Tobi's change was sufficient.

Comment (melindaloubser1) [reply pyproject.toml:210]: ah, got it - It was optimistic to hope that it would just work 😂

Comment (ancalita) [reply pyproject.toml]: Ok, the attempts this afternoon to use `sanic_testing` failed, every rewrite of `test_nlg()` failed :( So I have just changed the branch again in `pyproject.toml`.

**Pull request merged**